lib/roo_on_rails/checks/heroku/app_exists.rb

```diff
@@ -16,6 +16,10 @@ module Heroku
       # Output context:
       # - heroku.app.{env}: an app name.
       class AppExists < EnvSpecific
+        ACCEPTABLE_ENV_NAMES = {
+          'production' => ['prod', 'production'],
+          'staging'    => ['stg', 'staging']
```

Comment:
I guess whoever writes the code sets the convention—personally I'm fine either way.
I think it's not a huge deal as this will change when we change infra (DNS will be environment-local, so you won't need such suffixes).